### ISSUE TITLE
unexpected token "indent"

### ISSUE THREAD

**Issue by LuckyBlade** [vuejs/vue-loader#123]
After update from 7.4.2 to 7.5.2 I've got an error: 

ERROR in ./~/vue-html-loader!./~/template-html-loader?raw&engine=jade!./~/vue-loader/lib/selector.js?type=template&index=0!./vue/components/header/header.vue
Module build failed: Error: /home/***/vue/components/header/header.vue:2
    1| 
  > 2|   slot
    3|     header
    4|       router-view
unexpected token "indent" 

Error appears if there is an indent in template section:
```
<template lang = "jade">
  slot
    header
      router-view
</template>
```
And forks fine if there is no indent:
```
<template lang = "jade">
slot
  header
    router-view
</template>
```
In addition there is an error in script section in such code:
```
import param from "***";
  export default {
  }
```
And there is no error if : 
```
import param from "***";
export default {
}
```

**Comment by yyx990803** [comment]
This should be fixed in the next release.

**Comment by yyx990803** [comment]
fixed in 8.0.0
